package.json

```
@@ -18,9 +18,16 @@
     "postbuild": "npm run types && npm run bundlesize",
     "prepublishOnly": "npm run build"
   },
-  "main": "dist/react-tooltip.cjs.min.js",
-  "module": "dist/react-tooltip.esm.min.js",
+  "main": "dist/react-tooltip.cjs.js",
+  "module": "dist/react-tooltip.esm.js",
   "types": "dist/react-tooltip.d.ts",
+  "exports": {
+    ".": "./dist/react-tooltip.cjs.js",
+    "./dist/*.js": "./dist/*.js",
+    "./dist/react-tooltip.css": "./dist/react-tooltip.css",
+    "./dist/react-tooltip.d.ts": "./dist/react-tooltip.d.ts",
+    "./package.json": "./package.json"
+  },
```

Following my comment in #976, I fixed the problem using following configuration:

```suggestion
  "main": "dist/react-tooltip.cjs",
  "module": "dist/react-tooltip.mjs",
  "types": "dist/react-tooltip.d.ts",
  "exports": {
    ".": {
      "require": "./dist/react-tooltip.cjs",
      "import": "./dist/react-tooltip.mjs",
      "default": "./dist/react-tooltip.cjs"
    },
    "./dist/react-tooltip.css": "./dist/react-tooltip.css",
    "./package.json": "./package.json"
  },
```

I'm not sure if `"./package.json": "./package.json"` is useful tho

---

I got the `"./package.json": "./package.json"` from exports module on node, so, I'm not totally sure too

---

changes applied, new beta released, please test and let me know 🙏🏻 thanks!